trio/tests.py

```diff
@@ -7,7 +7,7 @@
 
 warn_deprecated(
     "trio.tests",
-    "0.24.0",
+    "0.22.1",
```

Comment:
I accidentally told @jakkdl to do this, not realizing this is for the version the function is deprecated in! (same for the other occurance).